modules/reindex/src/test/java/org/elasticsearch/index/reindex/RestReindexActionTests.java

```diff
@@ -41,10 +41,16 @@ public void testBuildRemoteInfoNoRemote() throws IOException {
     }
 
     public void testBuildRemoteInfoFullyLoaded() throws IOException {
+        Map<String, String> headers = new HashMap<>();
+        headers.put("first", "a");
+        headers.put("second", "b");
+        headers.put("third", null);
```

Comment:
This `null` is a leftover and I'll replace it with an empty string when I next switch to this branch.